packages/@aws-cdk/cli-lib/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@aws-cdk/cli-lib",
+  "description": "AWS CDK Programmatic CLI library",
+  "version": "0.0.0",
+  "private": true,
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "awslint": "cdk-awslint",
+    "build": "cdk-build",
+    "build+extract": "yarn build && yarn rosetta:extract",
+    "build+test": "yarn build && yarn test",
+    "build+test+extract": "yarn build+test && yarn rosetta:extract",
+    "build+test+package": "yarn build+test && yarn package",
+    "compat": "cdk-compat",
+    "lint": "cdk-lint",
+    "package": "cdk-package",
+    "pkglint": "pkglint -f",
+    "rosetta:extract": "yarn --silent jsii-rosetta extract",
+    "test": "cdk-test",
+    "watch": "cdk-watch"
+  },
+  "awslint": {
+    "exclude": []
+  },
+  "cdk-build": {
+    "env": {
+      "AWSLINT_BASE_CONSTRUCT": true
+    }
+  },
+  "author": {
+    "name": "Amazon Web Services",
+    "url": "https://aws.amazon.com",
+    "organization": true
+  },
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@aws-cdk/node-bundle": "0.0.0",
+    "@aws-cdk/cdk-build-tools": "0.0.0",
+    "@aws-cdk/pkglint": "0.0.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^14.18.32",
+    "ts-node": "^10.2.1",
+    "jest": "^27.5.1"
+  },
+  "dependencies": {
+    "aws-cdk": "0.0.0",
+    "@aws-cdk/core": "0.0.0",
+    "constructs": "^10.0.0"
```

Comment:
Ideally modern jsii packages don't have `dependencies` anymore, unless they are truly implementation details.

Reply:
Okay, so our Slack discussion aside, I've checked the other packages and they all include `constructs` as a dependency on main. However, when published it's moved to `devDependencies`.
For example:
https://github.com/aws/aws-cdk/blob/bf32cb138f30f4306c4e602f33ea3a52ca28263c/packages/@aws-cdk/aws-amplify/package.json#L104

I'll check if that's the case for this package as well.

Reply:
Confirmed. This would happen during the build.

Reply:
WUT. How the #%&# does that happen?

Reply:
`./scripts/transform.sh` is the entrypoint. But the tl;dr is that for alpha packages this script ends up doing the translation from `@aws-cdk/core` to `aws-cdk-lib` and moving dependencies to dev/peer deps.

To be clear, I don't _like_ how this is done at the moment but I'm reluctant to introduce sweeping changes with the repo restructuring on the horizon. Maybe that's a fallacy though. 🤔